_posts/2022-02-19-newpipe-0.22.0.md

```diff
@@ -0,0 +1,155 @@
+
+- [@litetex](https://github.com/litetex) implemented another improvement to error handling: If the app tried to open a file manager, but no SAF-compatible file manager was found, NewPipe just crashed. Now it tells you what went wrong and how to fix it.
+
+- Based on feedback from an accessibility-enabled user, [@litetex](https://github.com/litetex) stopped screen readers from telling you over and over and over again that there is a "Video preview thumbnail" when scrolling through lists.
+
+- We got another improvement for people with slow Internet or those facing a period of poor connection. The player used to "close with an error" (speak: crash) if the player took too long to buffer during a livestream. This happened because it just fell too far behind the edge of the livestream. To fix and improve this, [@TiA4f8R](https://github.com/TiA4f8R) implemented a two-fold change. The first was to increase the time period before the player is considered to have fallen too far behind, and the second was to simply drop the current play time and jump ahead to the edge of the livestream, skipping the un-buffered section entirely. No more player crashes! Never! Right guys?
+
+### Fixed
+
+- As the first entry in our big series of fixes this release, [@litetex](https://github.com/litetex) disabled an unreliable test for the SoundCloud stream extractor. Who even needs tests? (Just kidding, they're a cornerstone of good programming. ~~Eat your broccoli~~ Write your unit tests, kids.)
+
+- When a video link was opened with a timestamp URL parameter that had more than 3 digits, NewPipe ignored the 4th and higher digits. This meant that the app took you to the wrong timestamp. For example, 1001 seconds (16 minutes and 41 seconds) would be parsed as 1 second instead. [@mhmdanas](https://github.com/mhmdanas) has fixed this.
+
+- A nasty bug a lot of our users faced was that switching from one player type to another caused the playback to restart from the beginning. This has been fixed by [@litetex](https://github.com/litetex).
+
+- [@Stypox](https://github.com/Stypox) fixed the order of local search results from your search history, so now they are back to being ordered by creation date instead of alphabetical order (which made it seem like new entries weren't being added to the search result history database, unless you searched for a video titled "AAAA" or something).
+
+ -[@XiangRongLin](https://github.com/XiangRongLin) ensured that search history entries in tests are in the correct order, fixing the sometimes-failing-sometimes-passing CI pipeline.
```

Comment:
This is also development stuff.
```suggestion
- [@XiangRongLin](https://github.com/XiangRongLin) ensured that search history entries in tests are in the correct order, fixing the sometimes-failing-sometimes-passing CI pipeline.
```

Reply:
The idea behind this was to group them together (thats why its a subpoint), I will move it down.